rest/route/notification.go

```diff
@@ -60,107 +56,6 @@ func (h *notificationPostHandler) Run(ctx context.Context) gimlet.Responder {
 	return h.handler.Run(ctx)
 }
 
-///////////////////////////////////////////////////////////////////////
-//
-// POST /rest/v2/notifications/jira_comment
-
-type jiraCommentNotificationPostHandler struct {
-	APIJiraComment *model.APIJiraComment
```

Comment:
I think we can also get rid of this type and its service functions

Reply:
Good catch, removed.